src/main/java/uk/gov/companieshouse/model/PenaltyIdentifier.java

```diff
@@ -13,8 +13,8 @@
     private String companyNumber;
 
     @NotBlank(message = "penaltyReference must not be blank")
-    @Pattern(regexp = "([A-Z0-9/]{8,14})",
-        message = "penaltyReference is invalid")
+    @Pattern(regexp = "^[A-Z][0-9]{8}$|^PEN\\s?[1-2]A\\/[0-9]{8}$",
```

Comment:
The first part should be `^([A-Z][0-9]{7})$` and I do not think we need the $ or ^ sign in the middle of the regex.
To be honest I do not know about the \\ before the whitespace character and I presume one single escape sign should be fine `PEN ?[1-2]A\/[0-9]{8}`